config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml

```diff
@@ -241,7 +241,7 @@ periodics:
           cpu: 6
           memory: "16Gi"
 
-- cron: '1 22 * * 2,4,6' # Run at 14:01PST (22:01 UTC) on even days
+- cron: '1 22 * * 6' # Run at 14:01PST on Fri (22:01 UTC on Sat)
```

Comment:
The comment should say 'Sat' instead of 'Fri'.

Reply:
fixed